source/content/wordpress-cron.md

```diff
@@ -104,12 +104,10 @@ You can use external Crons if you want more control over your site's Cron jobs,
 
 ### Disable WP-Cron
 
-You will need to disable WP-Cron if your upstream does not have the definition below.
-
-Add the code below to your `wp-config.php` file to disable WP-Cron's internal processing:
+By default, Pantheon's WordPress upstreams have WP_CRON disabled. If you wish to continue using WP_CRON, you will need to add the code below to your `wp-config.php` file to enable WP-Cron's internal processing:
```

Comment:
@jspellman814 It'll be important to not the added line needs to be above the `require_once` of the `wp-config-pantheon.php` file.